main/acle.md

```diff
@@ -9382,6 +9415,2316 @@ are named after. All of the functions have external linkage.
+
+  __attribute__((arm_streaming))
+  svuint16_t svsqrshru[_u16_x4](svint64x4_t zn, uint64_t imm);
+  
+  ```
+
+#### SQRSHRUN
+
+Multi-vector saturating rounding shift right unsigned narrow and interleave
+
+``` c
+  
+  __attribute__((arm_streaming_compatible))
+  svuint16_t svsqrshrun[_u16_x2](svint32x2_t zn, uint64_t imm);
+  
+  
+  // Variants are also available for _u16
+  __attribute__((arm_streaming))
+  svuint8_t svsqrshrun[_u8_x4](svint32x4_t zn, uint64_t imm);
+  
+  ```
+
+#### SQDMULH (single)
+
+Multi-vector signed saturating doubling multiply high
+
+``` c
+  // Variants are also available for _single_s16_x2, _single_s32_x2
+  // and _single_s64_x2
+  __attribute__((arm_streaming))
+  svint8x2_t svsqdmulh[_single_s8_x2](svint8x2_t zdn, svint8_t zm);
+  
+  
+  // Variants are also available for _single_s16_x4, _single_s32_x4
+  // and _single_s64_x4
+  __attribute__((arm_streaming))
+  svint8x4_t svsqdmulh[_single_s8_x4](svint8x4_t zdn, svint8_t zm);
+  
+  ```
+
+#### SQDMULH (multi)
+
+Multi-vector signed saturating doubling multiply high
+
+``` c
+  // Variants are also available for _s16_x2, _s32_x2 and _s64_x2
+  __attribute__((arm_streaming))
+  svint8x2_t svsqdmulh[_s8_x2](svint8x2_t zdn, svint8x2_t zm);
+  
+  
+  // Variants are also available for _s16_x4, _s32_x4 and _s64_x4
+  __attribute__((arm_streaming))
+  svint8x4_t svsqdmulh[_s8_x4](svint8x4_t zdn, svint8x4_t zm);
+  
+  ```
+
+#### WHILEGE, WHILEGT, WHILEHI, WHILEHS, WHILELE, WHILELO, WHILELS, WHILELT
+
+While (resulting in predicate-as-counter). ``vl`` is expected to be 2 or 4.
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilege_c8(int64_t rn, int64_t rm, uint64_t vl);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilegt_c8(int64_t rn, int64_t rm, uint64_t vl);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilehi_c8(uint64_t rn, uint64_t rm, uint64_t vl);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilehs_c8(uint64_t rn, uint64_t rm, uint64_t vl);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilele_c8(int64_t rn, int64_t rm, uint64_t vl);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilelo_c8(uint64_t rn, uint64_t rm, uint64_t vl);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilels_c8(uint64_t rn, uint64_t rm, uint64_t vl);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svwhilelt_c8(int64_t rn, int64_t rm, uint64_t vl);
+  
+  ```
+
+#### WHILEGE, WHILEGT, WHILEHI, WHILEHS, WHILELE, WHILELO, WHILELS, WHILELT
```

Comment:
Repeated heading.  Just removing it should be OK, so that there's two blocks of functions under the same heading.